src/algorithms/calorimetry/CalorimeterHitReco.cc

```diff
@@ -106,6 +116,16 @@ void CalorimeterHitReco::AlgorithmChangeRun() {
 //------------------------
 void CalorimeterHitReco::AlgorithmProcess() {
 
+    // For some detectors, the cellID in the raw hits may be broken
```

Comment:
Wondering if this is after hcal hit merging. In juggler, that used to not leave the resulting hit with a valid cellID, but was modified over the summer to get the cellID of the highest energy hit. All hits should have a valid cellID.